packages/core/src/config/defaults.ts

```diff
@@ -0,0 +1,22 @@
+import { HealthCheck, TerminusOptions } from '@godaddy/terminus';
+
+const second = 1000;
+
+export const defaultHealthCheck = (isProduction: boolean, healthcheckOptions?: Partial<TerminusOptions['healthChecks']>) => ({
```

Comment:
What if you expand `healthCheckOptions` with `isProduction` so it's a single object? That way the first parameter (see example) eg. `false` isn't magical compared to `{ isProduction: false }`

Reply:
Good point!

Reply:
```
export const defaultHealthCheck = (
  healthcheckOptions: Partial<TerminusOptions['healthChecks']> & { isProduction: boolean; } = { isProduction: false }
) => ({
  verbatim: !healthcheckOptions.isProduction,
  __unsafeExposeStackTraces: !healthcheckOptions.isProduction,
  '/healthcheck': ((input) => {
    const { state } = input;
    if (state.isShuttingDown) {
      throw new Error('Server is shutting down. See ya!');
    }
  }) as HealthCheck,
  ...healthcheckOptions,
});
```